src/types/network.types.ts

```diff
@@ -40,6 +41,7 @@ export interface CustomNetworkType {
   skipFetchFromServer?: boolean;
   apiTimeout?: string;
   walletConnectV2ProjectId?: string;
+  walletConnectV2Options?: any;
```

Comment:
Record<string, any>; ?

Reply:
The proper way would be 
walletConnectV2Options?: WalletConnectV2Provider['options'];

but this will lead to multiple typescript issues due to the way WalletConnect is typed

For example:  Type 'WritableDraft<ICore>' is not assignable to type 'ICore'.
from redux because ICore is not "data", and thus doesn't belong in the Redux state

plus the classic

The inferred type of 'walletConnectV2OptionsSelector' cannot be named without a reference to '@walletconnect/sign-client/node_modules/@walletconnect/types'. This is likely not portable. A type annotation is necessary.

Since the options are for advanced cases only, and some options will be successfully ignored by WalletConnect even though they are present in typings, until the WalletConnect typing is updated from the package, I think this should be ok